manifests/package.pp

```diff
@@ -144,6 +145,23 @@
 
   }
 
+  $confd_dirs = join($confd_dir,',')
+
+  $params_vars= {
+    'EMBEDDED_RUBY'             => $use_embedded_ruby,
+    'LOG_LEVEL'                 => $log_level,
+    'LOG_DIR'                   => $log_dir,
+    'RUBYOPT'                   => $rubyopt,
+    'GEM_PATH'                  => $gem_path,
+    'CLIENT_DEREGISTER_ON_STOP' => true,
+    'CLIENT_DEREGISTER_HANDLER' => $deregister_handler,
+    'SERVICE_MAX_WAIT'          => $init_stop_max_wait,
+    'PATH'                      => $path,
+    'CONFD_DIR'                 => "${conf_dir},${confd_dir}",
+  }
```

Comment:
shouldn't heap size be here?

if we remove it, we would need need to bump the major version, which I'm not excited about doing since we'll be doing that soon enough for the next version of Sensu.

Reply:
heap_size afaik is used only on sensu enterprise, so it would be used in sensu::enterprise